.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: required
+
+language: generic
+
+services:
+  - docker
```

Comment:
As mentioned before, I believe docker is not necessary since you install it manually.

Reply:
OK seems to work 4734c8e2d6413a1f776a466bae78f1e34df6d340